components/pom.xml

```diff
@@ -43,5 +43,7 @@
     <modules>
         <module>org.wso2.carbon.micro.integrator.core</module>
         <module>org.wso2.carbon.micro.integrator.server</module>
+        <module>org.wso2.carbon.micro.integrator.security</module>
+        <!--<module>org.wso2.carbon.micro.integrator.extensions</module>-->
```

Comment:
Commented out module :)

Reply:
This will be fixed in https://github.com/wso2/micro-integrator/issues/10